scripts/itkdev-docker-compose

```diff
@@ -130,6 +216,14 @@ Configuration:
 EOF
 }
 
+# template:install must be run before we check for existence of
+# docker-compose.yml.
+if [[ "$#" -gt "0" && "$1" == "template:install" ]]; then
+  shift
+  template_install "$@"
+  exit
```

Comment:
This is actually also true for the traefik commands or you have to stand in folder with docker and .env file.